src/scripts/setup.sh

```diff
@@ -10,7 +10,7 @@ apt update -y
 apt install git nodejs openjdk-11-jre-headless -y
 apt install npm -y
 
-sudo -u mcuser -i << EOF
```

Comment:
Sorry, you still have this wrong. It's SUPPOSED to be `sudo -u mcuser -i << EOF`. That's why it's not working.

Reply:
And you ALSO need to leave `EOF` at the end of the file as is.

Reply:
lol I thought I had already done this. See most recent commit - I made that change.

Reply:
(at some point I lost the changes, but they are here on the branch now - if you could confirm that, that would be great)